server/parsec/components/postgresql/invite.py

```diff
@@ -362,14 +372,8 @@ async def _do_new_user_or_device_invitation(
 async def _human_handle_from_user_id(
     conn: AsyncpgConnection, organization_id: OrganizationID, user_id: UserID
 ) -> HumanHandle:
+    # TODO: implement this query and write the corresponding test
```

Comment:
This query seems to be used when sending invitation mails, we would need to implement it before deployment.